templates/ClientMethodOptionsSnippet.stencil

```diff
@@ -0,0 +1,23 @@
+/// User-configurable options for the `{{ model.clientName }}.{{ model.operationName }}` operation.
+public struct {{ model.name }}: AzureOptions {
+
+{% for property in model.properties %}
+    {{ property.comment }}
+    public let {{ property.name }}: {{ property.type }}{{property.defaultValue}}
```

Comment:
We need a "=" between the type and defaultValue {{ property.type }}{{property.defaultValue}}

Reply:
The `defaultValue` primitive is either "" or " = {value}"